apps/frontend/src/network/team/Routes.tsx

```diff
@@ -87,6 +87,9 @@ const Team: React.FC<{}> = () => {
                     <ToolModal
                       title="Add Link"
                       backHref={join(url, 'workspace')}
+                      onSave={(data: TeamTool) =>
+                        patch({ tools: [...(team.tools ?? []), data] })
```

Comment:
In this case I suppose it doesn't make a difference since the patch is idempotent, but should we check if a req is already pending before sending one? Or will this be solved in UI with a loading state that disables the button or so?

Reply:
I will sort this out when i get to loading :)